runelite-client/src/main/java/net/runelite/client/plugins/wintertodt/WintertodtPlugin.java

```diff
@@ -482,6 +489,25 @@ else if (numLogs == 0 && numKindling == 0 && currentActivity == WintertodtActivi
 		}
 	}
 
+	@Subscribe
+	public void onConfigChanged(ConfigChanged configChanged)
+	{
+		if (!configChanged.getGroup().equals("wintertodt"))
+		{
+			return;
+		}
+
+		if (configChanged.getKey().equals("showOverlay") && config.showOverlay())
+		{
+			overlayManager.add(overlay);
+		}
+
+		if (configChanged.getKey().equals("showOverlay") && !config.showOverlay())
+		{
+			overlayManager.remove(overlay);
+		}
```

Comment:
This could basically be
```suggestion
		if (configChanged.getGroup().equals("wintertodt") && configChanged.getKey().equals("showOverlay"))
		{
			if (config.showOverlay())
			{
				overlayManager.add(overlay);
			}
			else
			{
				overlayManager.remove(overlay);
			}
		}
```

Reply:
would it still need to` return` on an `else`?

Reply:
No.